File: Typescript/src/FizzBuzz.ts
export const FizzBuzz = (numbers: number) => {
  if (numbers % 3 === 0 && numbers % 5 === 0) {
    return "FizzBuzz";
  } else if (numbers % 3 === 0) {
    return "Fizz";
  } else if (numbers % 5 === 0) {
    return "Buzz";
  } else {
    return numbers;
  }
};

for (let i = 1; i <= 100; i++) {
  console.log(FizzBuzz(i));
}

File: Python/source/FizzBuzz.py
#!/usr/bin/python3

def FizzBuzz(number: int):
    if number % 15 == 0:
        return "FizzBuzz"
    elif number % 3 == 0:
        return "Fizz"
    elif number % 5 == 0:
        return "Buzz"
    else:
        return number


def main():
    for number in range(1, 101):
        print(FizzBuzz.FizzBuzz(number))



if __name__ == '__main__':
    main()
